bioconda_utils/bioconductor_skeleton.py

```diff
@@ -959,6 +959,7 @@ def sub_placeholders(x):
                 'build', OrderedDict((
                     ('number', self.build_number),
                     ('rpaths', ['lib/R/lib/', 'lib/']),
+                    ('run_exports', f'{{{{ pin_subpackage("bioconductor-{self.package_lower}", max_pin="x") }}}}'),
```

Comment:
Isn't it x.x here?

Reply:
I don't know. My gut feeling is that we with every release of BioC package devs have all freedom to deprecate functions etc ... so if I depend on deseq2 1.40 today I don't want to have deseq2 1.80 in a few years?

In other words, I do assume breakages in x.x releases with every BioC release.

Reply:
Keep in mind that I have no clue. And whatever you have done until now is better than what we have done so far.

Reply:
Thanks to @aliciaaevans for pointing me here. I just quickly want to chime in and second (or third) the decision to `max_pin="x.x"`. I previously went through the bioconductor developer docs and talked to a bioconductor package maintainer who basically confirmed what @bgruening is saying above. Namely, that every minor version bump can break anything, but that package versions from the same bioconductor release are assumed to be compatible with each other. For cross-reference, the original discussion is here:
https://github.com/bioconda/bioconda-recipes/issues/43905#issuecomment-1783290046

Reply:
`x.x` it is!